components/Map/Location/MapLocation.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect, useState } from 'react';
+import { useMapEvents, Marker, Popup } from 'react-leaflet';
+import L from 'leaflet';
+
+import { COORDS_EKATERINBURG } from 'common/constants/coords';
+
+export function MapLocation() {
+    const [position, setPosition] = useState<L.LatLng | null>(null);
+
+    const map = useMapEvents({
+        locationfound(e) {
+            if (!position) {
+                map.setView(e.latlng, map.getZoom());
+            }
+
+            setPosition(e.latlng);
+        },
+        locationerror(e) {
+            console.error(e);
+        },
+    });
+
+    useEffect(() => {
+        map.locate({ watch: true, maxZoom: 16 });
+
+        return () => {
+            map.stopLocate();
+        };
+    }, [map]);
+
+    return (
+        <Marker position={position || COORDS_EKATERINBURG}>
+            <Popup>Your location</Popup>
+        </Marker>
+    );
```

Comment:
Can you do `return null` here? I don't think that we need marker of user on this map

Reply:
Done

Reply:
+ remove unused imports